config/vaccines.yml

```diff
@@ -1,5 +1,25 @@
+# WARNING: Changing this configuration during the lifetime of a deployment may cause data in your application to become invalid.
+#          Please read the section below carefully, which details safe and unsafe changes to this file.
```

Comment:
Really great write-up here on the warning. Just one thing to add- it should be mentioned that if a change to the set of products is made, the "Guidance" tab of the import guidance in `public/Sara Alert Import Format.xlsx` would need to be updated accordingly

Reply:
[3d489db](https://github.com/SaraAlert/SaraAlert/pull/1225/commits/3d489db31a5ba3ff9a426316ea0e3f0aeaf17b42)